tools/ci_build/github/azure-pipelines/mac-ci-pipeline.yml

```diff
@@ -5,8 +5,9 @@ jobs:
   steps:
     - template: templates/set-test-data-variables-step.yml
     - script: |
+        sudo python3 -m pip install numpy==1.15.0
```

Comment:
Is this needed ?

numpy==1.15.0 is only for building redistributable packages.  This is not a packaging pipeline.

Reply:
This PR basically is to make sure the change did not break packaging pipeline. if changes cannot compile with old version, this place will be updated and at least we'll be informed about it and make corresponding changes in release pipelines.